.github/workflows/php.yml

```
@@ -28,7 +28,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['7.2', '7.3', '7.4']
+        php-versions: ['7.2', '7.3', '7.4', '8.0']
```

As per our conversations about supported versions, please remove unsupported version 7.2.
```suggestion
        php-versions: ['7.3', '7.4', '8.0']
```

---

I don't think we should remove 7.2
- It's supported for PHPUnit 8
- It doesn't slow down the overal test
- The more tests we can have for the current technical supported PHP version, the better

---

It's symbolic, we only officially support PHP versions that still are supported by the PHP Group. It isn't a question of performance.